definitions/variable/emissions/emissions.yaml

```
@@ -3,24 +3,27 @@
 # Gross emissions are only defined for CO2.
 
 - Emissions|{Level-1 Species}:
-    description: Emissions of {Level-1 Species}
+    description: Emissions of {Level-1 Species}, net of emissions to the atmosphere and the removal of {Level-1 Species} from the atmosphere.
```

I don't think that the description "Emissions of CO2, net of emissions to the atmosphere..." makes a lot of sense

---

It can be helpful to be explicit that we're talking about net emissions here. However, if the default assumption in the common definitions is that "Emissions" always means net, and we deal with gross emissions and removals some other way, then it could be more confusing than helpful to repeat the definition here (because it implies that the convention isn't actually a convention).

---

When I read "A, net of B", I think it means "A minus B" - that's what I meant with "doesn't make a lot of sense"...

---

Ah I see, I agree. What do you think of the below

```suggestion
    description: Emissions of {Level-1 Species} to the atmosphere. For the avoidance of doubt, these emissions are net emissions (i.e. include both positive (emission) and negative (removal) contributions. For positive contributions only, see "Gross Emissions|*").
```

---

Sounds good to me. How about adding that to the top-level, while changing 
"net of emissions"
to 
"the sum of sources and sinks" everywhere else?